### PR TITLE
Task 11 fibonacci

### PR DESCRIPTION
**Pull request by VladimirGOU**

### CODE REVIEW

**Review by TGacek89 — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
-// index.js
-// Hi this is Volodymyr ___
+const number = parseInt(prompt('Enter the number of terms: '));
+let n1 = 0, n2 = 1, nextTerm;
 
-//pull request
+console.log('Fibonacci Series:');
 
-alert("HELLO ITS MY FIRST COMMIT");
+for (let i = 1; i <= number; i++) {
```
Comment (VasylPosmit) [inline index.js:6]: We need a method to use

Comment (VasylPosmit) [reply index.js:6]: Otherwise it can not be tested or used

**Review by VasylPosmit — CHANGES_REQUESTED** [review]

**Review by celpjmnz — APPROVED** [review]

File: index.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
-// index.js
-// Hi this is Volodymyr ___
+const number = parseInt(prompt('Enter the number of terms: '));
+let n1 = 0, n2 = 1, nextTerm;
 
-//pull request
-
-alert("HELLO ITS MY FIRST COMMIT");
```
Comment (VasylPosmit) [inline index.js:6]: good that you've deleted unused code
It follows YAGNI

File: index.js [hunk 3]
```diff
@@ -1,6 +1,10 @@
...
+console.log('Fibonacci Series:');
+function fibonacci (number){for (let i = 1; i <= number; i++) {
+    console.log(n1);
+    nextTerm = n1 + n2;
+    n1 = n2;
+    n2 = nextTerm;
+}}
```
Comment (celpjmnz) [inline index.js:10]: Good function

Comment (VladimirGOU) [reply index.js:10]: Thanks)

**Review by TGacek89 — COMMENTED** [review]
Comment the fibonacchi on line 65

**Review by celpjmnz — APPROVED** [review]

**Review by TGacek89 — APPROVED** [review]

**Review by celpjmnz — APPROVED** [review]

**Pull request merged**